app/helpers/admin/dashboard_helper.rb

```diff
@@ -36,4 +36,13 @@ def relevant_account_timestamp(account)
 
     content_tag(:time, l(timestamp), class: 'time-ago', datetime: timestamp.iso8601, title: l(timestamp))
   end
+
+  def emergency_mode_notice
+    reason = EmergencyMode.reason
+    return if reason.blank?
+
+    content_tag(:div, class: 'flash-message') do
```

Comment:
Would it be worth using the `warning` class as well? (the default seems to just use a lighter grey, maybe this should be yellow/red?